share/poly.go

```diff
@@ -14,10 +14,16 @@ import (
 	"crypto/subtle"
 	"encoding/binary"
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 )
 
+type Suite interface {
```

Comment:
This looks like a good example of a more minimalist package-specific Suite interface working as expected...